src/renderer/views/Playlist/Playlist.js

```diff
@@ -420,6 +434,27 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    getDurationPlaylistItems: function () {
+      const modifiedPlaylistItems = deepCopy(this.playlistItems)
+      for (const v in modifiedPlaylistItems) {
```

Comment:
Maybe just use `forEach` instead of having to write `modifiedPlaylistItems[v]` in the loop

Reply:
done!